cmd/oras/blob/cmd.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package blob
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"oras.land/oras/cmd/oras/internal/option"
+)
+
+func Cmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "blob [command]",
+		Short: "[Preview] Blob operations",
+	}
+
+	cmd.AddCommand(
+		deleteCmd(),
+	)
+	return cmd
+}
+
+func deleteCmd() *cobra.Command {
+	var opts deleteBlobOptions
+	cmd := &cobra.Command{
+		Use:   "delete <name@digest> [flags]",
+		Short: "[Preview] Delete a blob from a remote registry",
+		Long: `[Preview] Delete a blob from a remote registry
+** This command is in preview and under development. **
+
+Example - Delete blob:
+  oras blob delete localhost:5000/hello@sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5
+
+Example - Delete blob from the insecure registry:
+  oras blob delete localhost:5000/hello@sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5 --insecure
+
+Example - Delete blob from the HTTP registry:
+  oras blob delete localhost:5000/hello@sha256:9a201d228ebd966211f7d1131be19f152be428bd373a92071c71d8deaf83b3e5 --plain-http        
```

Comment:
Suggest to remove this example: localhost will be automatically parsed as plain http registry